server/controller/answers.js

```diff
@@ -0,0 +1,22 @@
+const Question = require('../models/question.js');
+const Answer = require('../models/answer.js');
+
+exports.addAnswer = async (ctx) => {
+  await Question.findById(ctx.request.body._id)
```

Comment:
This would totally depend on what we send from the app. We would receive the objectId from the DB then change it on the front end to `question_id` and then send it back to query the DB. Leaving it as `_id` just means we would change the name less. If you want to make it more explicit we could create like `const questionId = ctx.request.body._id`,